src/MDMTransition.h

```diff
@@ -37,6 +37,17 @@ NS_SWIFT_NAME(Transition)
 @end
 
 /**
+ Transition with custom duration overrides the default transition duration
+ */
+NS_SWIFT_NAME(TransitionWithCustomDuration)
+@protocol TransitionWithCustomDuration
+/**
+ Return the duration for the transition
+ */
+- (NSTimeInterval)transitionDuration:(nonnull id<MDMTransitionContext>)context;
```

Comment:
Objective-C naming's a bit different from Swift; you generally have to be explicit about the argument names like so: `transitionDurationWithContext:`